packages/docs/docs/guide/markdown.md

```diff
@@ -121,6 +121,9 @@ Rendering of TOC can be configured using the [`markdown.toc`](../config/README.m
 
 ## Custom Containers <Badge text="default theme"/>
 
+Containers can be defined by their types, titles, and contents. The default titles would be applied while customized titles were missing or empty.
```

Comment:
```suggestion
Custom containers can be defined by their types, titles, and contents.
```

I'm not sure I understand the second sentence. Are you saying that customized titles are sometimes missing or empty?

Reply:
Yes. Intentionally or unintentionally. Maybe some people wonder why there is a title there if I didn't leave a title there. This sentense can clarify that.
This issue was raised because some people missed the docs about custom title. So I think some simple explanation about default will be user friendly too.

Reply:
Got it. I think you achieve this by giving new dedicated section headings to default vs custom titles below, so if we want additional clarity, we should include it in those sections rather than in this intro part since we don't have any code examples yet.